orchagent/intfsorch.cpp

```diff
@@ -1362,9 +1363,13 @@ bool IntfsOrch::isRemoteSystemPortIntf(string alias)
     Port port;
     if(gPortsOrch->getPort(alias, port))
     {
+
+        if (port.m_type == Port::LAG)
+            return(port.m_system_lag_info.switch_id != gVoqMySwitchId);
```

Comment:
Can we put the statement in {} to be consistent with the rest of the change?

Reply:
o.k. I'll fix this.

Reply:
Fixed